.eslintrc.js

```diff
@@ -18,7 +18,6 @@ module.exports = {
 		node: true,
 	},
 	rules: {
-		'require-await': 'error',
```

Comment:
Thoughts on removing this @wjrosa ?

This rule was causing issues with using code like:

```
eceButton.on( 'shippingaddresschange', async ( event ) =>
	shippingAddressChangeHandler( api, event, elements )
);
```

Even though the await was within `shippingAddressChangeHandler`. I think this rule might be too restrictive so I decided to remove it.

Happy to discuss and go with another alternative :)

Reply:
I'm not 100% sure (this isn't my strong suit) but I think another alternative is to remove the async keyword. 

eg 

```js
eceButton.on( 'shippingaddresschange', ( event ) =>
	shippingAddressChangeHandler( api, event, elements )
);
``` 

The linting error is being thrown because the callback function is defined as async but that function doesn't use the `await` keyword directly. There's really 3 options to appease the linting error. 

1. Remove this rule. Though I'm not sure if that's a good idea. 
2. Remove the async keyword. (See example above).
3. Add an await. See example below.

```js
eceButton.on( 'shippingratechange', async ( event ) =>
	await shippingRateChangeHandler( api, event, elements )
);
```

Reply:
Adding the `await` keyword still triggers the error `async functions are only available in ES8` on the `async` keyword. What if we also mirror our ESlint rules with WCPay? I see they have some different versions for the ESLint packages, and they use the `override` key on .eslintrc.js

Reply:
I have updated our ESLint version (and the min requirement on .jshintrc) in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3436/commits/bc581aa7fe0a73f2c698d39a1082ca87497c749a . Tests are still working and I could not see any issues with it. Let me know if you think that's a good move or if it is better to revert it 👍

Reply:
I have reverted this rule removal as I fixed all the warnings in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3436/commits/1e080897c523f5776b7523a96d4bd78c9d7890f4